diff --git a/Footer.js b/Footer.js
index e76610fb43f0fb5a472a8623f3f304dbec70f2fd..00838fc1112d7e67165d7f7fbccfe51a66f4435e 100644
--- a/Footer.js
+++ b/Footer.js
@@ -68,9 +68,9 @@ export const Footer = memo(forwardRef((props, ref) => {
                 React.createElement("div", { className: cx(fr.cx("fr-footer__content"), classes.content) },
                     contentDescription !== undefined && (React.createElement("p", { className: cx(fr.cx("fr-footer__content-desc"), classes.contentDesc) }, contentDescription)),
                     React.createElement("ul", { className: cx(fr.cx("fr-footer__content-list"), classes.contentList) }, [
-                        "legifrance.gouv.fr",
-                        "gouvernement.fr",
+                         "info.gouv.fr",
                         "service-public.fr",
+                      "legifrance.gouv.fr",
                         "data.gouv.fr"
                     ].map((domain, i) => (React.createElement("li", { className: cx(fr.cx("fr-footer__content-item"), classes.contentItem), key: i },
                         React.createElement("a", { className: cx(fr.cx("fr-footer__content-link"), classes.contentLink), target: "_blank", href: `https://${domain}`, title: `${domain} - ${t("open new window")}` }, domain))))))),
