diff --git a/lib/index.js b/lib/index.js
index 763fd3c51587ca2331e95f136eecf7cea7134df7..8eafae4e41462ef6c4315dc49b608c5aadbbb818 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -192,7 +192,7 @@ const renderGlyphs = (ctx, glyphs, positions, x, y) => {
     const encodedPositions = positions.map((pos, i) => ({
         xAdvance: pos.xAdvance * scale,
         yAdvance: pos.yAdvance * scale,
-        xOffset: pos.xOffset,
+        xOffset: pos.xOffset * scale,
         yOffset: pos.yOffset,
         advanceWidth: glyphs[i].advanceWidth * advanceWidthScale,
     }));
